samples/algorithms/repeat-until-success/RepeatUntilSuccess.qs

```diff
@@ -0,0 +1,110 @@
+namespace Microsoft.Quantum.Samples.RepeatUntilSuccess {
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Math;
+    open Microsoft.Quantum.Preparation;
+
+    /// Example of a Repeat-until-success circuit
+    /// The circuit is executed on a "target" qubit using an "ancilla" and 
+    /// "resource" qubit.
+    /// The goal is to measure Zero for both the ancilla and resource qubit.
+    /// If this succeeds, the circuit will have effectively applied an 
+    /// Rz(arctan(2)) gate (also known as V_3 gate) on the target qubit.
+    /// If this fails, rerun the circuit up to <limit> times.
+    @EntryPoint()
+    operation ApplyRzArcTan2(
+        inputValue : Bool,
+        inputBasis: Pauli,
+        limit: Int
+    ) : (Bool, Result, Int) {
+        using ((ancilla, resource, target) = (Qubit(), Qubit(), Qubit())) {
+            // Prepare qubits
+            PrepareXZero(ancilla);
```

Comment:
The API documentation comment for `PrepareXZero` indicates that it expects its input to be in the |1⟩ state, but here, `ancilla` starts off in |0⟩ as it is freshly allocated. In general, the Q# verb "Prepare" indicates that an operation expects a qubit input to be in the |0⟩ state, so this may be an issue with the name and documentation comments for `PrepareXZero`.

Reply:
What naming would you suggest in this case?
Would `SetXZeroFromOne` work?